easybuild/easyblocks/c/cp2k.py

```diff
@@ -845,6 +855,8 @@ def sanity_check_step(self):
         }
         if self.cfg['library']:
             custom_paths['files'].append(os.path.join('lib', 'libcp2k.a'))
+            custom_paths['files'].append(os.path.join('include', 'libcp2k.h'))
+            custom_paths['files'].append(os.path.join('include', 'libcp2k.mod'))
```

Comment:
In the [CP2K ticket](https://github.com/cp2k/cp2k/issues/53) they said it's enough to have these files, for my Fortran user who asked for this this wasn't the case however. I think a few more mod files are required but rather than track them down I'm sticking with the naive approach of putting them all in the include dir. If someone else knows better I'm open to suggestions